### ISSUE TITLE
More than one device not receving message

### ISSUE THREAD

**Issue by kbramprasath** [redsolution/xabber-android#365]
Hello sir,

            Same user logged in three device(User name "A"), if a message is send to this user(User name "A") from another user(User name: "B") only one devices is recceiving the messages other two devices are not receiving. Kindly provide me a solution.

**Comment by TameOfGroans** [comment]
Message sent to a particular resource of user_A?

**Comment by kbramprasath** [comment]
yes, but only one device is receiving data rest of them not receiving, but they also logged in same login, need solution for this issue

**Comment by andrewnenakhov** [comment]
This could possibly be due to different priorities on connected resources. Post more details on connected resources, otherwise it could be that xmpp works like it should according to standard.

**Comment by matrixx740** [comment]
I have the same issue. The problem is, the messages are sent to and only to the resource with highest priority, even when incoming messages are received from a resource with lower priority. Resource of incoming messages is not disclosed and messages cannot be addressed properly.
